examples/README.md

```diff
@@ -23,49 +23,59 @@ trunk serve --release
 We're also publicly hosting the examples at `https://examples.yew.rs/<EXAMPLE>`.
 As an example, check out the TodoMVC example here: <https://examples.yew.rs/todomvc>
 
-## List of examples
+## List of Examples
 
- - CT - Type of most components , "F" for function components and "S" for struct components, "SF" for a mix of both.
+- `Example`: Link and name of each example.
+- `CT`: Type of most components
+  - `F` for function components
+  - `S` for struct components
+  - `SF` for a mix of function and struct components.
+- `Description`: Short sentance on what the example demonstrates.
```

Comment:
I don't think it's necessary to explain this column. "sentance" is also misspelled either way.